lib/event_relay/events.ex

```diff
@@ -16,11 +16,15 @@ defmodule ER.Events do
   end
 
   def from_events_for_topic(topic_name: topic_name) do
-    table_name = ER.Events.Event.table_name(topic_name)
+    table_name =
+      topic_name
+      |> ER.Events.Event.table_name()
```

Comment:
I would put the downcase in the table_name method I think.

Reply:
That was my original plan, let me take another look.